### PR TITLE
feat(PSR): ⌚ Add PSR Clock interface as dependency

### PR DESCRIPTION
**Pull request by nickvergessen**
- Server PR at https://github.com/nextcloud/server/pull/35872
- OCP PR so it Psalm finds the interface https://github.com/nextcloud-deps/ocp/pull/4

### CODE REVIEW

**Comment by ChristophWurst** [comment]
For?

**Comment by nickvergessen** [comment]
Server PR at https://github.com/nextcloud/server/pull/35872

**Review by ChristophWurst — APPROVED** [review]

File: composer/installed.json [hunk 1]
```diff
@@ -6942,6 +6993,6 @@
             "install-path": "../web-auth/webauthn-lib"
         }
     ],
-    "dev": false,
+    "dev": true,
```
Comment (come-nc) [inline composer/installed.json:6996]: Should not be pushed

File: composer/installed.php [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
         'pretty_version' => 'dev-master',
         'version' => 'dev-master',
-        'reference' => '9ef27e13d670f2afe61c08f9662731e727ff00ee',
+        'reference' => '33a6ef1c2223e8d105fa05b6b83663dcddab8109',
         'type' => 'library',
         'install_path' => __DIR__ . '/../',
         'aliases' => array(),
-        'dev' => false,
+        'dev' => true,
     ),
```
Comment (come-nc) [inline composer/installed.php:11]: same

**Review by come-nc — CHANGES_REQUESTED** [review]

File: composer/installed.json (hunk 1 above)
Comment (nickvergessen) [reply composer/installed.json]: Could be fixed by `composer install --no-dev`

Thanks for the hint

**Review by come-nc — APPROVED** [review]

**Pull request merged**